debugAdapterProtocol.json

```diff
@@ -93,9 +93,10 @@
 					"message": {
 						"type": "string",
 						"description": "Contains the raw error in short form if `success` is false.\nThis raw error might be interpreted by the client and is not shown in the UI.\nSome predefined values exist.",
-						"_enum": [ "cancelled" ],
+						"_enum": [ "cancelled", "notStopped" ],
 						"enumDescriptions": [
-							"request was cancelled."
+							"the request was cancelled.",
+							"the request may be retried once the adapter is in a 'stopped' state."
```

Comment:
Talking about the request being retried sounds a bit off, it may be that this _exact_ request will not be what the client wants to retry, just that they may want to make that type of request again later. Like `this request can only be made when the adapter is in a 'stopped' state.`

Reply:
What do you get from cdp in this case, would you adopt this flag in js-debug?

Reply:
I don't have a need for this _currently_ in js-debug, but could in the future depending on how memoryReferences shake out...